ci/jenkinsfiles/release.groovy

```diff
@@ -0,0 +1,335 @@
+    }
+    stage('Notify promotion start on slack') {
+      steps {
+        script {
+          String message = "Starting release ${VERSION} from build ${env.RC_VERSION}: ${BUILD_URL}"
+          pipelineLib.setSlackBuildStatus("${SLACK_CHANNEL}", "${message}", 'gray')
+        }
+      }
+    }
+    stage('Set Labels') {
+      steps {
+        container('maven') {
+          script {
+            pipelineLib.setLabels()
+          }
+        }
+      }
+    }
+    stage('Setup') {
+      steps {
+        container('maven') {
+          script {
+            pipelineLib.setup()
+            sh 'env'
+          }
+        }
+      }
+    }
+    stage('Fetch Release Candidate') {
+      steps {
+        container('maven') {
+          sh "git fetch origin 'refs/tags/v${RC_VERSION}*:refs/tags/v${RC_VERSION}*'"
+        }
+      }
+    }
+    stage('Checkout') {
+      steps {
+        container('maven') {
+          script {
+            pipelineLib.gitCheckout("v${RC_VERSION}")
+          }
+        }
+      }
+    }
+    stage('Update Version') {
+      steps {
+        container('maven') {
+          script {
+            pipelineLib.updateVersion("${VERSION}")
+          }
+        }
+      }
+    }
+    stage('Compile') {
+      steps {
+        container('maven') {
+          script {
+            pipelineLib.compile()
+          }
+        }
+      }
+    }
+    stage('Linting') {
+      steps {
+        container('maven') {
+          script {
+            pipelineLib.lint()
+          }
+        }
+      }
+    }
+    stage('Run Unit Tests') {
+      steps {
+        script {
+          def stages = [:]
+          stages['backend'] = pipelineLib.runBackEndUnitTests()
+          stages['frontend'] = pipelineLib.runFrontEndUnitTests()
+          parallel stages
+        }
+      }
+    }
+    stage('Publish') {
+      when {
+        allOf {
+          not {
+            branch 'PR-*'
+          }
+          not {
+            environment name: 'DRY_RUN', value: 'true'
+          }
+        }
+      }
+      environment {
+        MESSAGE = "Release ${VERSION}"
+        TAG = "v${VERSION}"
+      }
+      stages {
+        stage('Git Commit and Tag') {
+          steps {
+            container('maven') {
+              script {
+                pipelineLib.gitCommit("${MESSAGE}", '-a')
+                pipelineLib.gitTag("${TAG}", "${MESSAGE}")
+              }
+            }
+          }
+        }
+        stage('Publish Coldstorage Package') {
+          steps {
+            container('maven') {
+              script {
+                echo """
+                  -------------------------------------------------
+                  Upload Coldstorage Package ${VERSION} to ${CONNECT_PROD_URL}
+                  -------------------------------------------------
+                """
+                if (DRY_RUN_RELEASE == 'false') {
+                  pipelineLib.uploadPackage("${VERSION}", 'connect-preprod', "${CONNECT_PROD_URL}")
```

Comment:
Is the key `connect-preprod` accurate, as we publish the release to connect prod.

Reply:
I will update this part.